app/src/main/java/org/simple/clinic/newentry/PatientEntryEvent.kt

```diff
@@ -71,3 +72,5 @@ data class ReminderConsentChanged(val reminderConsent: ReminderConsent) : Patien
 data class AlternativeIdChanged(val identifier: Identifier) : PatientEntryEvent()
 
 data class InputFieldsLoaded(val inputFields: InputFields) : PatientEntryEvent()
+
+data class ColonyOrVillagesFetched(val colonyOrVillages: List<String>?) : PatientEntryEvent()
```

Comment:
The `List` will never be `null`. We can make the `getColonyOrVillages` as non-null. If there is no data in the db, Room database will return an empty list instead of a nullable object for lists. So we don't have to handle nullable here.